Formula/p/pre-commit.rb

```diff
@@ -9,7 +9,7 @@ class PreCommit < Formula
   head "https://github.com/pre-commit/pre-commit.git", branch: "main"
 
   bottle do
-    rebuild 1
+    rebuild 2
```

Comment:
The bottle block should not be touched in PRs. It's only automatically generated.

Reply:
ok. fixed. thanks